libp2p/src/tutorials/ping.rs

```diff
@@ -181,15 +169,15 @@
 //!
 //! #[async_std::main]
 //! async fn main() -> Result<(), Box<dyn Error>> {
-//!     env_logger::init();
```

Comment:
Logger is gone now but still mentioned in the tutorial.

Reply:
3717d1fe2